app/models/consistency.php

```diff
@@ -0,0 +1,85 @@
+<?php
+namespace Transvision;
+
+// Build arrays for the search form, ignore mozilla.org
+$channel_selector = Utils::getHtmlSelectOptions(
+    array_diff($repos_nice_names, ['mozilla.org']),
+    $repo,
+    true
+);
+
+$target_locales_list = Utils::getHtmlSelectOptions(
+    Project::getRepositoryLocales($repo),
+    $locale
+);
+
+/*
+    Since this is a very heavy view for Aurora (default repo), we display
+    an introductory text when reaching the page and use a parameter to
+    determine if we actually want to display the strings.
+*/
+$display_results = isset($_GET['display']);
+$reference_locale = Project::getReferenceLocale($repo);
+
+// Set a default for the number of strings to display
+$strings_number = 0;
+
+// We are going to show the placeholder text and avoid any calculation
+if (! $display_results) {
+    return;
+}
+
+// Get non-empty strings for reference and locale in this repository
+$source_strings = array_filter(Utils::getRepoStrings($reference_locale, $repo));
+$target_strings = array_filter(Utils::getRepoStrings($locale, $repo));
+
+if (! empty($source_strings)) {
+    $duplicated_strings_english = Consistency::findDuplicates($source_strings);
+
+    // Filter English duplicates before comparing them with the localization
+    $duplicated_strings_english = Consistency::filterStrings($duplicated_strings_english, $repo);
+
+    // Get the strings duplicated in English but not in the localization
+    $missing_duplicates = array_diff_key(
+        $duplicated_strings_english,
+        Consistency::findDuplicates($target_strings)
+    );
```

Comment:
I was trying to understand why a string supposed to be duplicated would only display one result in the translation column, and realized that the logic is flawed and I need to find a completely different approach.

Example: there are two strings for 'closed' in English, but one is translated as 'chiusa', the other one as 'chiuso', so they're not duplicated in Italian.